boot/environ/include/efi/ProcessorBind.h

```diff
@@ -147,7 +147,7 @@ typedef char                CHAR8;
 ///
 /// 1-byte signed value.
 ///
-typedef char                INT8;
+typedef signed char         INT8;
```

Comment:
Why is this needed?
Clang defaults to unsigned char or?

Reply:
It's signed char in psdk\basetsd.h:191 => redefinition with different type error.